TM1py/Services/ProcessService.py

```diff
@@ -264,12 +264,13 @@ def execute(self, process_name: str, parameters: Dict = None, timeout: float = N
 
     @require_version(version="11.3")
     def execute_process_with_return(self, process: Process, timeout: float = None, cancel_at_timeout: bool = False,
-                                    **kwargs) -> Tuple[bool, str, str]:
+                                    return_async_id: bool = False, **kwargs) -> Tuple[bool, str, str]:
```

Comment:
FYI, typing is incorrect as the async_id is str afaik

Reply:
Yeah. The return type hint is incorrect when `return_async_id` is passed as True. 
However, the default case has the correct type hint.
I will take a look at this in a separate PR.